src/app/index.ts

```diff
@@ -56,13 +57,21 @@ const app = express();
 
 app.use(compression()); // Compress all routes
 // helps mitigate cross-site scripting attacks and other known vulnerabilities
+const reportOnly =
+  EXPRESS_CONTENT_SECURITY_POLICY_CONFIG['report-only'] || EXPRESS_CONTENT_SECURITY_POLICY_CONFIG.reportOnly;
+if (reportOnly !== undefined) {
+  delete EXPRESS_CONTENT_SECURITY_POLICY_CONFIG['report-only'];
+  delete EXPRESS_CONTENT_SECURITY_POLICY_CONFIG.reportOnly;
+}
+
 app.use(
   helmet({
     // override default contentSecurityPolicy directive like script-src to include cloudflare cdn and github static content
     // might consider turning this off to allow individual front-ends set Content-Security-Policy on meta tags themselves if list grows long
     // <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline'  https://cdnjs.cloudflare.com;" >
     contentSecurityPolicy: {
       directives: EXPRESS_CONTENT_SECURITY_POLICY_CONFIG,
+      reportOnly: !!reportOnly,
```

Comment:
There is a 'danger' to using this header IMO. If it's accidentally set to true it negates all the benefits of csp (`Content-Security-Policy-Report-Only` header is set instead of `Content-Security-Policy`). At the very worst both csp headers should co-exist (i.e `Content-Security-Policy` and `Content-Security-Policy-Report-Only`). I would vote to have this directive removed (defaults to false) or to set both as described here https://github.com/helmetjs/helmet/issues/351#issuecomment-1015498560

Reply:
cool, I will remove the reportOnly for now, I think we can use the more general express_reponse_headers to append other headers including the `report-only` variant